101-cosmosdb-free/README.md

```diff
@@ -0,0 +1,10 @@
+# Create a free-tier Azure Cosmos DB account for Core (SQL) API
```

Comment:
Need to add the badges, see the sample readme or the pipeline task will have the exact MD you need to add.  AzDO pipelines are broken for PS Core error reporting so the task succeeds even though failures are detected.

Reply:
Added